make/deps.mk

```diff
@@ -18,20 +14,16 @@ DEPS = amqp_client \
 	exml \
 	fcm \
 	folsom \
-	fs_event \
-	fs_sync \
 	gen_smtp \
 	getopt \
 	gproc \
 	hep \
-	horse \
```

Comment:
Isn't this being used by `kz_{term,binary,json,map,time}_tests`?

Reply:
Only if you specifically run `make perf` - which most folks are not doing. There's a comment on the `dep_horse` line to that effect.